HOWTO_RELEASE.md

```diff
@@ -0,0 +1,18 @@
+Release procedure is as follows:
```

Comment:
Can you add a header to this document? Currently Contributing.md, the Readme, and others all have headers. Perhaps even adding `# Releasing Gitea` followed by two line breaks to the top of the file (or even "Release Procedure").